eth_typing/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .misc import (
```

Comment:
I think we should find a different directory name. It's probably inspired by the `eth_typing` in the `py-evm` project but the reason that is called `eth_typing` there is because it is really intended to be moved out of the project to become a package of shared types that are used across various ethereum projects.

In this particular case though, I think we just need a place to put various types for *this* project. I would vote for creating a directory `typing` directly under `eth_pm`

Reply:
@cburgdorf Yup, was inspired by `py-evm`, but your suggestion makes sense to me.